applications/GeoMechanicsApplication/tests/cpp_tests/custom_constitutive/test_incremental_linear_elastic_interface_law.cpp

```diff
@@ -33,54 +36,95 @@ namespace Kratos::Testing
 
 KRATOS_TEST_CASE_IN_SUITE(LinearElasticLawForInterfacesHas2DWorkingSpace, KratosGeoMechanicsFastSuiteWithoutKernel)
 {
-    auto law = GeoIncrementalLinearElasticInterfaceLaw{};
+    auto law = GeoIncrementalLinearElasticInterfaceLaw{std::make_unique<InterfacePlaneStrain>()};
 
     KRATOS_EXPECT_EQ(law.WorkingSpaceDimension(), 2);
 }
 
+KRATOS_TEST_CASE_IN_SUITE(LinearElasticLawForInterfacesHas3DWorkingSpace, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    auto law = GeoIncrementalLinearElasticInterfaceLaw{std::make_unique<InterfaceThreeDimensionalSurface>()};
+
+    KRATOS_EXPECT_EQ(law.WorkingSpaceDimension(), 3);
+}
```

Comment:
Perhaps we can merge this new test case with the previous one and let the name reflect that the working space dimension is determined by its constitutive law dimension. For instance:
```suggestion
KRATOS_TEST_CASE_IN_SUITE(WorkingSpaceOfLinearElasticLawForInterfacesDependsOnLawDimension, KratosGeoMechanicsFastSuiteWithoutKernel)
{
    auto law = GeoIncrementalLinearElasticInterfaceLaw{std::make_unique<InterfacePlaneStrain>()};
    KRATOS_EXPECT_EQ(law.WorkingSpaceDimension(), 2);

    law = GeoIncrementalLinearElasticInterfaceLaw{std::make_unique<InterfaceThreeDimensionalSurface>()};
    KRATOS_EXPECT_EQ(law.WorkingSpaceDimension(), 3);
}
```

If you agree with this suggestion, then I would propose to propagate that pattern to the other unit tests here, too. If you don't agree, I'd suggest to use better test case names, to reflect in which context the linear elastic law is being applied (i.e. plane strain or three-dimensional).

Reply:
2D and 3D tests are joint.